src/analytics/redux/tracked_events.js

```diff
@@ -111,20 +196,10 @@ function handleBlockMovedByPosition( clientIds, toIndex ) {
 
 export const trackedEvents = {
 	'core/block-editor': {
-		insertBlock( blocks ) {
-			trackBlocksHandler(
-				blocks,
-				'editor_block_inserted',
-				( { name } ) => ( { block_name: name } )
-			);
-		},
-		insertBlocks( blocks ) {
-			trackBlocksHandler(
-				blocks,
-				'editor_block_inserted',
-				( { name } ) => ( { block_name: name } )
-			);
-		},
+		insertBlock: trackBlockInsertion,
```

Comment:
I noticed that the `insertBlock` Redux action is actually invoking `insertBlocks`. Could we simply track events for the latter?

https://github.com/WordPress/gutenberg/blob/d27c43fda419995a80b1cbdeafe2b24d9a0e2164/packages/block-editor/src/store/actions.js#L541-L548

Reply:
Thanks for the suggestion! I'm currently checking why `insertBlocks` doesn't work but `insertBlock` does 🙃

Reply:
So I had to keep both, it only tracks the first dispatched action, so if `insertBlock` is dispatched, that's the action that will be tracked and not the `insertBlocks` that's called within.

I updated here https://github.com/wordpress-mobile/gutenberg-mobile/pull/5976/commits/effe376cb59ea829723d0d8538e4bfc8731441db let me know what you think 😄

Reply:
Ah, ok. If only the first dispatched action is processed let's keep both.